score_keeper_client/src/index.js

```diff
@@ -3,12 +3,18 @@ import ReactDOM from 'react-dom/client';
 import './index.css';
 import App from './App';
 import { BrowserRouter } from 'react-router-dom'
+import { Auth0Provider } from "@auth0/auth0-react";
 
 const root = ReactDOM.createRoot(document.getElementById('root'));
 root.render(
-  <BrowserRouter>
-    <App />
-  </BrowserRouter>
+  <Auth0Provider
+    domain="chandan-sm.us.auth0.com"
```

Comment:
You should add all sensitive details API_KEYS/Domains/clientID in `.env` and it should not be available publicly, Add `domain` and `clientId` to `.env`.

Reply:
Changes made as suggested